packages/react-instantsearch-dom-geo/src/Connector.js

```diff
@@ -9,14 +9,15 @@ export class Connector extends Component {
   static propTypes = {
```

Comment:
Why don't we have the other options [like in InstantSearch.js](https://community.algolia.com/instantsearch.js/v2/widgets/geoSearch.html#usage)?

Reply:
We do have the other options when it make sense. The only option that is missing is `paddingBoundingBox` because I didn't manage to implement it properly. The rest of the options is either implemented as component (`Redo`, `Control`)
 or can be achieved with other widgets (`Configure`, `ClearRefinements`).